.github/workflows/enos-run.yml

```diff
@@ -15,6 +15,7 @@ on:
 
 env:
   PKG_NAME: boundary
+  DOCKER_MIRROR: docker.mirror.hashicorp.services
```

Comment:
Is this env var used by enos anywhere? It gets used by the unit tests [here](https://github.com/hashicorp/boundary/blob/28756dce6717ccbac3bfc0c11ba77d1bbac18047/testing/dbtest/docker/Makefile#L9) and [here](https://github.com/hashicorp/boundary/blob/28756dce6717ccbac3bfc0c11ba77d1bbac18047/internal/credential/vault/supported.go#L40-L44).

Reply:
Oh 😂 I thought it was some environment variable that got magically consumed. I'll take a look.